.github/prep-hotfix-sources.sh

```diff
@@ -1,12 +1,14 @@
-#!/bin/bash -x
+#!/bin/bash -xe
 
 HOTFIX_TAG=$1
 REPO_PATH=$2
 SPEC_PATH=$3
 
 # Create OSG build dirs
 build_dir=$PWD/_build_dir
-mkdir -p "$build_dir/{osg,upstream}"
+for subdir in osg upstream; do
+    mkdir -p "$build_dir/$subdir"
+done
```

Comment:
```suggestion
mkdir -p "$build_dir"/{osg,upstream}
```

I don't remember the history here, i am sorry if i misled you into this, but basically for the brace expansion to work you do not quote the braces.

Reply:
Argh, no, that's just an oversight on my part. Good catch